tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/cassandra/CassandraTableManager.java

```diff
@@ -216,12 +216,12 @@ protected TableName createImmutableTableName(TableHandle tableHandle)
         );
     }
 
-    private void executeQueryIgnoreError(String sql)
+    private void executeQueryIgnoreTypeError(String sql)
     {
         try {
             queryExecutor.executeQuery(sql);
         }
-        catch (IllegalStateException e) {
+        catch (CassandraQueryExecutor.TypeNotSupportedException e) {
```

Comment:
This is a part of `QueryExecutor` logic. I don't thinkg `CassandraTableManager` should know what types `QueryExecutor` supports. Especially while the output of `executeQuery` is not used by `CassandraTableManager`.

I could return nothing from `CassandraQueryExecutor.executeQuery` as well but I wanted to keep it similar to other `QueryExecutors`, if anyone needed it outside of `CassandraTableManager`.